languages/kotlin/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,133 @@
+Kotlin is **statically typed** programming language developed by JetBrains. This means that type of variables is defined on compile-time.
+
+## Entry point
+
+To run your Kotlin program you need to define so-called entry point: function with name `main` with or without arguments. However, in Exercism course we are using automatic tests that are using other functions defined in exercises.
+
+Entry point with arguments (you can use them for building CLI applications):
+
+```kotlin
+fun main(args: Array<String>) {
+    println("Hello, Exercism!")
+    println("Program args are: $args")
+}
+```
+
+Or ignore arguments completely:
+
+```kotlin
+fun main() {
+    println("Hello, Exercism!")
+}
+```
```

Comment:
Is this information needed for the student to solve the exercise? I suspect it isn't, in which case this information should be removed. In general, the `introduction.md` document should only contain information that is needed to understand the fundamentals of the concept and solve the exercise should be provided (see [this description](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)). You could consider having this information in the `after.md` document though.

Reply:
You are right. It's not for exercise but it's kinda important thing to know to be able to say that language is "mastered". Moving it to `after` is a great idea. Thanks.